test/commands/auth/device/login.test.ts

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/* eslint-disable @typescript-eslint/ban-ts-ignore */
```

Comment:
If we're doing this in both of the test files, we should move it to the .eslintrc in the test dir